interfaces/builtin/raw_usb.go

```diff
@@ -35,7 +35,7 @@ const rawusbConnectedPlugAppArmor = `
 /dev/bus/usb/[0-9][0-9][0-9]/[0-9][0-9][0-9] rw,
 
 # Allow access to all ttyUSB devices too
-/dev/ttyUSB[0-9]* rw,
+/dev/tty{USB|ACM|AMA}[0-9]* rw,
```

Comment:
That's a different set of devices than the one that was here before the branch got force pushed to. IIRC ttyAMA is the ARM UART controller, so it's wired to the internal bus on the SoC rather than the USB.

Reply:
Yes I updated branch as I was told by  @alfonsosanchezbeato that some USB modem devices come as ttyAMA. @alfonsosanchezbeato can you confirm please.

Reply:
Right, ttyAMA* devices are UART ones, I think we can drop those, sorry for the noise.